cluster/swarm/cluster.go

```diff
@@ -221,8 +221,17 @@ func (c *Cluster) createContainer(config *cluster.ContainerConfig, name string,
 
 	c.scheduler.Unlock()
 
-	log.WithFields(log.Fields{"NodeName": n.Name, "NodeID": n.ID}).Debugf("Scheduling container %s to ", name)
 	container, err := engine.CreateContainer(config, name, true, authConfig)
+	containerFlag := name
+	if containerFlag == "" && err == nil {
+		containerFlag = stringid.TruncateID(container.ID)
+	}
+
+	if err != nil {
+		log.WithFields(log.Fields{"NodeName": n.Name, "NodeID": n.ID}).Errorf("Failed to create container")
```

Comment:
`Errorf("Failed to create container")` -> `Errorf("Failed to create container: %v", err)`